whenipress.d.ts

```diff
@@ -0,0 +1,49 @@
+declare module 'whenipress' {
+    type Handler = (event?: Event) => void;
+
+    type WhenIPressPlugin = Partial<{
+      mounted: (globalInstance: PendingKeyboardEventManager) => void;
+      bindingRegistered: (binding: string[], globalInstance: PendingKeyboardEventManager) => void;
+      bindingStopped: (binding: string[], globalInstance: PendingKeyboardEventManager) => void;
+      allBindingsStopped: (globalInstance: PendingKeyboardEventManager) => void;
+      beforeBindingHandled: (binding: string[], globalInstance: PendingKeyboardEventManager) => boolean | void;
+      afterBindingHandled: (binding: string[], globalInstance: PendingKeyboardEventManager) => void;
+    }>
+
+    type WhenIPressPluginOptions = Partial<{
```

Comment:
I don't think we need this part. Its only referenced in the documentation and not used anywhere in the source code.

Reply:
But this is for the developer using this library when they're writing their own plugin with individual hooks like `mounted`, `bindingRegistered` etc.

Reply:
Understood. Just wonder what the purpose of `urlsToSkip` and `skipAllUrls` is here though? That isn't a global plugin option, its just an example of options that a plugin owner might choose to include.

Am I missing something?

Reply:
So they're entirely custom options defined by the developer using the library and they can just as well use anything for options like `{ foo: true, bar: false }` etc? Just need to be clear :sweat_smile:

Reply:
Indeed. I probably need to clear that up in the documentation. Its just a unified area for providing configuration options to user's of your plugin.

Reply:
Ok, cool. Thanks for clearing it up. I'll push a new commit now.